pyanaconda/network.py

```diff
@@ -48,6 +48,8 @@
 from pyanaconda.regexes import HOSTNAME_PATTERN_WITHOUT_ANCHORS
 
 from gi.repository import NetworkManager
+# # Register the types so that the use in threads is safe
```

Comment:
You have here one '#' more than it's needed. It looks to me, like copy-paste error.